store/settings/index.js

```diff
@@ -101,9 +101,10 @@ function Settings (state, emitter) {
   })
 }
 
-Settings.prototype.loadTabs = function (state, emitter) {
-  require('./tabs/storage')(state, emitter)
+Settings.prototype.initTabs = function (state, emitter) {
   require('./tabs/search')(state, emitter)
+  require('./tabs/ui')(state, emitter)
+  require('./tabs/storage')(state, emitter)
```

Comment:
We should really do e.g. `this.storage = require('./tabs/storage')(state, emitter)` instead of doing `state.settings.storage = ...` in the function.

Reply:
Or perhaps `this.tabs.storage = ..` is better.